apps/web/app/(authenticated)/(app)/app/apis/[apiId]/settings/update-api-name.tsx

```diff
@@ -26,25 +27,39 @@ type Props = {
 export const UpdateApiName: React.FC<Props> = ({ api }) => {
   const { toast } = useToast();
   const { pending } = useFormStatus();
+  const router = useRouter();
+  const updateName = trpc.apiSettings.updateName.useMutation({
+    onSuccess: (_data) => {
```

Comment:
if we don't use it, we can just remove it
```suggestion
    onSuccess: () => {
```